### DISCUSSION TITLE
Updating Perplexica when new update comes.

### DISCUSSION THREAD

**Discussion by DanielIsThinking** [ItzCrazyKns/Vane#526] [Q&A] [answered]
Hey everyone sorry if this question comes out as dumb, but I read the whole main page and could not find a reference to it, when Perplexica is updated do I have to do anything (I'm on Linux btw using the Docker process of installation) or once installed it will update automatically?

**Comment by delfireinoso** [comment]


**Reply by DanielIsThinking** [reply]
Thanks!

**Comment by MinhHaDuong** [comment, marked as answer]
cd /opt/perplexica
docker-compose pull
docker-compose up -d

as per the update guide: https://github.com/ItzCrazyKns/Perplexica/blob/master/docs/installation/UPDATING.md
